go/common/tribool/tribool_test.go

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) 2024 Fantom Foundation
+//
+// Use of this software is governed by the Business Source License included
+// in the LICENSE file and at fantom.foundation/bsl11.
+//
+// Change Date: 2028-4-16
+//
+// On the date above, in accordance with the Business Source License, use of
+// this software will be governed by the GNU Lesser General Public License v3.
+
+package tribool
+
+import "testing"
+
+func TestTribool(t *testing.T) {
+	tests := []struct {
```

Comment:
nit: make it a `map[string]struct{...}` and put the name of the test case as a key in the map

Reply:
somehow this style is better understand by my IDE and I can run individual tests (perhaps it is more standard?). Does not matter in this case, but sometimes it is convenient. 

<img width="549" alt="image" src="https://github.com/Fantom-foundation/Carmen/assets/7114574/a94afdfe-9dc1-425f-836c-f33160e5a6d4">

Reply:
That's pretty cool ... k, in that case, go ahead.